src/constants/initial.js

```diff
@@ -2,12 +2,22 @@
 export const initial = {
   interest: {
     update: false,
-    data: { topic: "", id: -1, visibility: true },
+    data: {
+      topic: "",
+      id: -1,
+      priority: 1,
+      visibility: true
+    },
     links: []
   },
   achievement: {
     update: false,
-    data: { achievement: "", id: -1, visibility: true },
+    data: {
+      achievement: "",
+      id: -1,
```

Comment:
What is this for?

Reply:
This has initial default data for all the empty forms, eariler `priority` field was missing at multiple places, this is for those missing `priorty` fields.